docs/generated/notification-services/alertmanager.md

```diff
@@ -43,7 +43,7 @@ You should turn off "send_resolved" or you will receive unnecessary recovery not
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: <config-map-name>
+  name: argo-rollouts-notification-configmap
```

Comment:
generated by running `make docs`

These were recently renamed upstream to a `argocd` specific name :/